src/repl.js

```diff
@@ -1,8 +1,13 @@
 /* eslint-disable no-console */
 
 const inquirer = require('inquirer')
-const chalk = require('chalk')
 const gradient = require('gradient-string')
+let chalk
+
+async function chalkModule() {
```

Comment:
Hello @danielsous!

What do you think about importing chalk dependency at the top of the file? Is there a reason to import inside a function?

```
import chalk from 'chalk';

console.log(chalk.bold(coolGradient('\nHerbs 🌿 - Interative REPL')))
```

Reply:
Hi @maikvortx !

Well, Herbs2REPL is entirely in CJS, and the Chalk package was updated in which it became an ESM-only package, with that we lost the possibility of importing it through require(), and as the change of Herbs2REPL from CJS to ESM would not be a good idea, since it would break the projects that are using it at this moment, the ideal was to use import() to be able to import the chalk lib converting it from ESM to CJS. And the way to do that would be using await import(), and it needs to be inside an asynchronous function.


Note: I can't just use the `import chalk from 'chalk'` because it asks me to change the package to ESM.

> import chalk from 'chalk'
> ^^^^^^
> 
> SyntaxError: Cannot use import statement outside a module

Reply:
@danielsous why did you need to update chalk? We can not leave this old version?

Reply:
@jhomarolo Well, Chalk was already in version 5.0.1 in which there was this change. Basically, I'm solving a dependency problem, so we can always keep a more up-to-date version in use. I can go back to version 4.1.2, but in that case we won't be able to update Chalk while the project is in CJS. What do you think?

Reply:
I don't see why it is relevant to `Herbs Shell` issue. Let´s keep it on version 4.1.2 for now and open a issue to discuss about upgrading Chalk version or replacing it.

Reply:
I believe we can leave chalk in 4.1.2 until we discuss this topic in another issue.

Reply:
I changed the version to 4.1.2 of Chalk